runtime/amd64.S

```diff
@@ -679,14 +679,13 @@ G(caml_system__frametable):
 #elif defined(SYS_mingw64) || defined(SYS_cygwin)
         .section .rdata,"dr"
 #else
-        .section    .rodata.cst8,"a",@progbits
+        .section    .rodata.cst16,"aM",@progbits,16
 #endif
-        .globl  G(caml_negf_mask)
         .align  SIXTEEN_ALIGN
+        .globl  G(caml_negf_mask)
 G(caml_negf_mask):
         .quad   0x8000000000000000, 0
         .globl  G(caml_absf_mask)
-        .align  SIXTEEN_ALIGN
```

Comment:
I like to be very explicit about alignment.  Please don't remove this `.align` even if it happens to be redundant.